### DISCUSSION TITLE
x11 keymap not work in kitty

### DISCUSSION THREAD

**Discussion by OriLiMu** [kovidgoyal/kitty#6957] [Q&A]
I add these code in `.Xmodmap` file. Which will map key `Capslock` to key `F10`. 
```sh
clear lock
keycode  66 = F10
```
It works fine in alacritty but not work in kitty. How to fix it?

**Comment by kovidgoyal** [comment]
kitty uses libxkbcommon you can see what key events it receives by using --debug-input most likely your xmodmap is not working with it.

**Reply by OriLiMu** [reply]
Not key event captured when start with `--debug-input`. I can see the mouse event in it but when I press capslock nothing happens. Is there a way to solve it?
I want map capslock to f10 or run a script.

**Reply by kovidgoyal** [reply]
No clue, I dont use xmodmap you can simply remap capslock however you like in kitty.conf see the kitty faq
